build_test/lib/src/package_reader.dart

```diff
@@ -62,3 +54,25 @@ class PackageAssetReader implements AssetReader {
   Future<String> readAsString(AssetId id, {Encoding encoding: UTF8}) =>
       _resolve(id).readAsString(encoding: encoding);
 }
+
+/// Returns all assets that match [glob] in [package] with a [path].
+///
+/// **INTERNAL ONLY**.
+Iterable<AssetId> globAssets(String package, String path, Glob glob) {
+  if (package == null) {
```

Comment:
These are overkill, they aren't marked as optional args so if somebody wants to pass in nulls it is fine for it to blow up.